srcpkgs/Kooha/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'Kooha'
+pkgname=Kooha
+version=1.2.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gettext glib-devel"
+makedepends="glib-devel libadwaita-devel"
```

Comment:
We don't need `glib-devel` and `libadwaita-devel` here.

Reply:
You are right